pkg/host-timesync-daemon/Dockerfile

```diff
@@ -1,16 +1,9 @@
 FROM linuxkit/alpine:0c069d0fd7defddb6e03925fcd4915407db0c9e1 AS mirror
 
 RUN apk add --no-cache go musl-dev git
-ENV GOPATH=/go PATH=$PATH:/go/bin GO111MODULE=off
+ENV GOPATH=/go PATH=$PATH:/go/bin
 
-ENV VIRTSOCK_COMMIT=f1e32d3189e0dbb81c0e752a4e214617487eb41f
-RUN mkdir -p $GOPATH/src/github.com/linuxkit && \
-  cd $GOPATH/src/github.com/linuxkit && \
-  git clone https://github.com/linuxkit/virtsock.git && \
-  cd virtsock && \
-  git checkout $VIRTSOCK_COMMIT
-
-COPY .  /go/src/host-timesync-daemon
```

Comment:
The move to go modules made this able to just import it, and all of this build stuff goes away?

Reply:
It's vendored at that commit. It could have been done without modules. Not sure why we did it this way originally.

Reply:
In any case, we now have it in `go.sum`, so that should be good enough.